plugin/src/main/groovy/com/btkelly/gnag/reporters/BaseExecutedViolationDetector.groovy

```diff
@@ -27,12 +27,12 @@ abstract class BaseExecutedViolationDetector extends BaseViolationDetector {
     abstract void executeReporter()
 
     protected final ReporterExtension reporterExtension
-    protected final ReportHelper reportHelper;
+    protected final ProjectHelper reportHelper;
```

Comment:
A wise man once told me not to trust IntelliJ refactoring no matter how good it may seem.

![bitmoji](https://render.bitstrips.com/v2/cpanel/9941525-120874784_10-s1-v1.png?transparent=1&palette=1&width=246)